proto/umee/leverage/v1beta1/query.proto

```diff
@@ -27,6 +27,27 @@ service Query {
     option (google.api.http).get = "/umee/leverage/v1beta1/borrowed";
   }
 
+  // BorrowedValue queries for the usd value of the borrowed amount of a user
+  // by token denomination.  If the denomination is not supplied, the sum across
+  // all borrowed tokens isreturned.
+  rpc BorrowedValue(QueryBorrowedValueRequest) returns (QueryBorrowedValueResponse) {
+    option (google.api.http).get = "/umee/leverage/v1beta1/borrowed_value";
+  }
+
+  // Lent queries for the lent amount of a user by token denomination.
+  // If the denomination is not supplied, the total for each lent token is
+  // returned.
+  rpc Lent(QueryLentRequest) returns (QueryLentResponse) {
```

Comment:
nit: can we call this loaned* instead? Applies to all new queries. WDYT?

Reply:
Applied universally (with some wording changes)